src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -87,19 +80,18 @@ public CommandResult execute(Model model) throws CommandException {
     }
 
     /**
-     * Creates and returns a {@code Person} with the details of {@code personToEdit}
+     * Creates and returns a {@code person} with the details of {@code personToEdit}
```

Comment:
Instead of {@code person}, it should be {@code Person} because 'Person' is the class name.